gui/doc/tree.csv

```diff
@@ -1,4 +1,6 @@
 ﻿Name,Type,Default,Doc
 expanded,dynamic(bool|str[]),True,"if boolean value False: only one node can be expanded at one given level else it should be an array of the ids that need to be expanded"
 multiple,bool,FALSE,"If True, the user can select multiple items by holding the Ctrl key while clicking on items."
+children_only_selection,bool,False,"If True, the user can only select leaf nodes"
```

Comment:
select_leafs_only?
enable_non_leaf_selection?
all_nodes_selectable?
only_leafs_selectable?

Reply:
I'll go for `select_leafs_only`